docs/specs/bind-unbind-workflow.md

```diff
@@ -0,0 +1,22 @@
+## Bind / Unbind New Work flow
+
+The goal of this new work flow is to give more control over the actions that the database takes on bind and unbind to the APB author. This means that we want the apb to be launched on on bind and unbind. 
```

Comment:
Also what is the *new* workflow? because right now I thought we had a setting that if enabled will actually make a call to the apbs to do a bind (whatever that bind wants to do). If unset, we return what's in etcd probably from a previous provision call.